packages/eslint-config-commons-ui/package.json

```diff
@@ -27,7 +27,7 @@
     "eslint-plugin-jsx-a11y": "^6.7.1",
     "eslint-plugin-markdown": "^3.0.0",
     "eslint-plugin-playwright": "^0.12.0",
-    "eslint-plugin-prettier": "^5.0.0",
+    "eslint-plugin-prettier": "^4.2.1",
```

Comment:
Why?

Reply:
This caused linting to fail.
We're using `prettier 2.8.8` and there seem to be some incompatibility issues based on https://github.com/prettier/eslint-plugin-prettier/issues/562.

Reply:
Why are we using prettier 2.x if 3.x is already out?

Reply:
How is this resolved?

Reply:
This will be resolved in #527